packages/loopring_v3/contracts/core/impl/libexchange/ExchangeBlocks.sol

```diff
@@ -249,6 +249,10 @@ library ExchangeBlocks
                 );
                 uint txDataOffset = 0;
 
+                if (approved && txType != ExchangeData.TransactionType.WITHDRAWAL) {
```

Comment:
`!=` change to `==`?

Reply:
If it's a withdrawal the token transfer still needs to be done regardless if it's approved or not. If a withdrawal is approved like this we just don't check the normal approval methods.